PF/Assignment 4/Day4Assignment.java

```diff
@@ -0,0 +1,96 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Day4Assignment
+{
+	final int NUMBER_OF_COLUMNS = 4; // for matrix of 4X4 for N Queen Problem
+	List<String> towerStrings = new ArrayList<String>();
```

Comment:
variable towerStrings is only suppose to use in method towerOfHanoi(), so make the variable local rather than instance variable, to avoid memory spacing issue.